sickrage/ws/MedusaWebSocketHandler.py

```diff
@@ -0,0 +1,48 @@
+# coding=utf-8
+# Author: p0psicles
+# URL: http://code.google.com/p/sickbeard/
+#
+# This file is part of SickRage.
+#
+# SickRage is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SickRage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with SickRage. If not, see <http://www.gnu.org/licenses/>.
+
+
+import json
+import time
+from tornado.websocket import WebSocketHandler
+
+clients = []
+
+
+def push_to_web_socket(msg):
```

Comment:
This is the function that can be included and used to write to the ws. In this case i'm sending a notification. But you could already start sending other jsons.